pkgs/applications/editors/vim/plugins/vim-utils.nix

```diff
@@ -488,34 +488,37 @@ rec {
 
   toVimPlugin =
     drv:
-    drv.overrideAttrs (oldAttrs: {
-      name = "vimplugin-${oldAttrs.name}";
-      # dont move the "doc" folder since vim expects it
-      forceShare = [
-        "man"
-        "info"
-      ];
-
-      nativeBuildInputs =
-        oldAttrs.nativeBuildInputs or [ ]
-        ++ lib.optionals (stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
-          vimGenDocHook
+    drv.overrideAttrs (
+      finalAttrs: oldAttrs: {
+        name = "vimplugin-${oldAttrs.name}";
+        # dont move the "doc" folder since vim expects it
+        forceShare = [
+          "man"
+          "info"
         ];
 
-      doCheck = oldAttrs.doCheck or true;
+        nativeBuildInputs =
+          oldAttrs.nativeBuildInputs or [ ]
+          ++ lib.optionals (stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
+            vimGenDocHook
+          ];
 
-      nativeCheckInputs =
-        oldAttrs.nativeCheckInputs or [ ]
-        ++ lib.optionals (stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
-          vimCommandCheckHook
-          # many neovim plugins keep using buildVimPlugin
-          neovimRequireCheckHook
-        ];
+        doCheck = oldAttrs.doCheck or true;
 
-      passthru = (oldAttrs.passthru or { }) // {
-        vimPlugin = true;
-      };
-    });
+        nativeCheckInputs =
+          oldAttrs.nativeCheckInputs or [ ]
+          ++ lib.optionals (stdenv.buildPlatform.canExecute stdenv.hostPlatform) (
+            [
+              vimCommandCheckHook
```

Comment:
Note that `vimCommandCheckHook` is written to also use `neovim-unwrapped`; check line 429.

Reply:
Hmm... might need to get updated to use vim instead for the goal of this work?

Reply:
I introduced vimCommandCheck and I am using neovim. vimCommandCheck is applied mostly to neovim plugins I think. Anyway replacing neovim by vim would have a similar effect as neovim wouldn't be able to use plugins when vim breaks.
I dont think it's worth it to provide vimPlugins sets just to run the test. Your best bets are either: 
- to fix neovim on your platform such that everyone benefits from it
- override `neovim = vim` in the hook ?
- disable the checks